themes/edx.org/cms/templates/widgets/sock.html

```diff
@@ -16,7 +16,7 @@
   <div class="wrapper-inner wrapper">
     <section class="sock" id="sock">
       <header>
-        <h2 class="title sr">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h2>
+        <h2 class="title sr sr-only">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h2>
```

Comment:
I am pretty sure you don't need both sr and sr-only. In platform, we encourage using sr-only from now on.